web/static/js/views/cards/show.js

```diff
@@ -3,13 +3,21 @@ import { connect }        from 'react-redux';
 import CardModal          from '../../components/cards/modal';
 import Actions            from '../../actions/current_card';
 
+
 class CardsShowView extends React.Component {
   componentDidMount() {
     const { dispatch, params } = this.props;
 
-    console.log(params);
 
-    dispatch(Actions.showCard(params.id[0], params.id[1]));
+    dispatch(Actions.showCard(this.getCard(params.id[1])));
+  }
+
+  getCard(id) {
+    let cId = +id;
```

Comment:
Just a couple of things...
- Can you move this function just after the `componentWillMount` one? I'm following the convention where React lifecycle functions are always placed at the top, so it's easy to know what's going on.
- Can you add it a '_' prefix? I always add it to private functions so I can see private functionality at first sight :)

Thanks!